src/Faker/Provider/fa_AF/Company.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Faker\Provider\fa_AF;
+
+class Company extends \Faker\Provider\Company
+{
+    protected static $formats = array(
+        '{{companyName}}',
+    );
+
+    /**
+     * @source Afghanistan Kankor (University Entrance Exam) 2003-2017 data.
```

Comment:
not enough, you must add a link to a source I can verify. same for other formatters with the same source

Reply:
I collected the Kankor data for my PhD research purposes, and still I did not publish these data anywhere. But I add a link to one of my paper that confirms it.

Reply:
There is no way to check that the data actually comes from there, as it's not published in your paper...

Reply:
I mean the content of the paper confirms and verifies the Kankor data collection ...

Reply:
I need a link to the actual collection.

Reply:
Link added ...